lib/oli/accounts/schemas/user.ex

```diff
@@ -191,6 +204,9 @@ defmodule Oli.Accounts.User do
         false
     end
   end
+
+  defp is_age_verification_enabled(_changeset),
```

Comment:
Do we need to pass the `changeset` in if we never use it?

Reply:
We need since we are chaining this function in the validations as well